insights/parsers/openshift_get.py

```diff
@@ -61,6 +73,15 @@ def metadata_name_items(data):
     return {item['metadata']['name']: item for item in data['items']}
 
 
+@parser('oc_get_node')
+class OcGetNode(YAMLParser):
+    """Class to parse ``oc get node -o yaml --all-namespaces``"""
+
+    def get_node(self):
+        """ dict: Returns a dictionary of openshift nodes information."""
+        return metadata_name_items(self.data)
```

Comment:
@shzhou12 if you used the `@property` decorator and renamed the method you could make this a bit cleaner and more pythonic, maybe something like:
```
@property
def nodes(self):
    return metadata_name_items(self.data)
```
You could also consider making this an attribute by overloading the `parse_content` method and adding it there.  This applies to all of the classes in this module.

Reply:
Hi @bfahr, is it OK that only update these @property methods (and remove the old non-@property ones) in the 3.x branches? and in this 1.x we can keep these old (and not @property) methods?

Reply:
@xiangce yes that would be OK with me.

Reply:
@bfahr - Thanks for your advice, @shzhou12 has updated the code and document according to your comments. I'm going to merge it.